docs/api.md

```diff
@@ -178,10 +182,16 @@ class DummyCompositeAlgo(tools.CompositeAlgo):
         predictions = 0
         return predictions
 
-    def load_model(self, path):
+    def load_head_model(self, path, rank):
         return json.load(path)
 
-    def save_model(self, model, path):
+    def save_head_model(self, model, path, rank):
+        json.dump(model, path)
+
+    def load_trunk_model(self, path, rank):
+        return json.load(path)
+
+    def save_trunk_model(self, model, path, rank):
         json.dump(model, path)
```

Comment:
Should we also illustrate that the rank should be serialized as well?

Reply:
It does not have to be serialized.
The rank could be used to change the serializer depending on its value but won't be serialized.

Reply:
Ok, I guess it's much harder to illustrate use cases where this would be relevant in a "dummy" example. Let's leave it as it is then.

Reply:
Exactly.